content/docs/guide.md

```diff
@@ -68,10 +68,26 @@ If none of the above work for you, try running it in a container
 from our Fedora based image. It contains packit installed from `main` branch,
 i.e. the same you'd get by pip installing from Github.
 
-    $ podman run -ti --rm -v $PWD:/src --security-opt label=disable quay.io/packit/packit bash
+    $ podman run -ti --rm -v $PWD:/src:z quay.io/packit/packit bash
     $ packit
     Usage: packit [OPTIONS] COMMAND [ARGS]...
 
+Depending on the command you want to perform you need to mount secrets and
+configuration files, like in the following examples:
+
+#### packit copr-build
+
+    $ podman run -ti --rm -v ~/.config/copr:/root/.config/copr:z -v $PWD:/src:z quay.io/packit/packit bash
+    $ packit copr-build
+
+#### packit build
+
+    $ podman run -ti --rm -v ~/.ssh/:/root/.ssh:z -v $PWD:/src:z quay.io/packit/packit bash
+    $ export KRB5CCNAME=/tmp/ccache
+    $ fkinit -u username
```

Comment:
Sorry, I just noticed this. It would be nice to make it more explicit, something like:

```suggestion
    $ fkinit -u <Fedora Account username>
```

Or if you have a better idea, please use that. The important would be to make it clear that something needs to be substituted here.